src/cli/commands/test/cache.rs

```diff
@@ -21,6 +31,52 @@ pub mod cache {
 		pub hash: String,
 	}
 
+	fn is_valid_cairo_contract(contract_path: &PathBuf) -> Result<(), CacheError> {
```

Comment:
i've been scourging different codebases, however as far as i know, they usually use `full path` without really checking the extension because in the end

one of the example is `https://docs.rs/csv/latest/csv/struct.Reader.html`

i don't have any strong opinions and its easy to do either way so i'm cool with it!